app/boards/shields/dumbpad/Kconfig.defconfig

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if SHIELD_DUMBPAD || SHIELD_DUMBPAD_OLED_TOP
+
+config ZMK_KEYBOARD_NAME
+	default "DumbPad"
+
+if ZMK_DISPLAY
+
+config I2C
+	default y
+
+config SSD1306
+	default y
+
+config SSD1306_REVERSE_MODE
+	default y
+
+endif # ZMK_DISPLAY
+
+if LVGL
+
+config LVGL_HOR_RES_MAX
+	default 128
+
+config LVGL_VER_RES_MAX
+	default 32
+
```

Comment:
These lines should be removed as of #1352

Reply:
Done.